### PR TITLE
patch(*): remove null option from required fields on psvs and hgv

### PR DESCRIPTION
**Pull request by naathanbrown**
## Ticket title

Can't accept null if it is required. Number of wheels driven is an exception 

## Changelog

- remove null property from required fields on psvs

### CODE REVIEW

File: json-definitions/v3/tech-record/get/psv/complete/index.json [hunk 1]
```diff
@@ -85,9 +85,6 @@
       "anyOf": [
```
Comment (gjulien-bjss) [inline json-definitions/v3/tech-record/get/psv/complete/index.json:85]: you'll have to remove the `anyOf` if you only have one option in the array, or it'll fail validation

File: json-definitions/v3/tech-record/get/psv/complete/index.json [hunk 2]
```diff
@@ -156,9 +153,6 @@
           "type": "integer",
           "minimum": 0,
           "maximum": 99
-        },
```
Comment (gjulien-bjss) [inline json-definitions/v3/tech-record/get/psv/complete/index.json:159]: same with the `anyOf` here

File: json-definitions/v3/tech-record/get/psv/complete/index.json (hunk 1 above)
Comment (naathanbrown) [reply json-definitions/v3/tech-record/get/psv/complete/index.json:85]: Cheers, working out what the hell is wrong with this PR atm tho, hate your bot (which is my fault)

File: json-definitions/v3/tech-record/get/psv/complete/index.json [hunk 3]
```diff
@@ -711,8 +705,7 @@
     },
     "techRecord_brakes_dtpNumber": {
       "type": [
-        "string",
-        "null"
+        "string"
       ],
```
Comment (gjulien-bjss) [inline json-definitions/v3/tech-record/get/psv/complete/index.json:709]: nitpick: can move it out of an array and just have "type": "string"

File: json-definitions/v3/tech-record/get/psv/testable/index.json [hunk 4]
```diff
@@ -128,9 +127,6 @@
           "type": "integer",
           "minimum": 0,
           "maximum": 99
-        },
-        {
-          "type": "null"
```
Comment (gjulien-bjss) [inline json-definitions/v3/tech-record/get/psv/testable/index.json:133]: same with the `anyOf` here

File: json-definitions/v3/tech-record/put/hgv/complete/index.json [hunk 5]
```diff
@@ -358,14 +358,12 @@
     },
     "techRecord_applicantDetails_address1": {
       "type": [
-        "null",
```
Comment (gjulien-bjss) [inline json-definitions/v3/tech-record/put/hgv/complete/index.json:361]: can move out of the array

File: json-definitions/v3/tech-record/put/psv/complete/index.json [hunk 6]
```diff
@@ -79,9 +79,6 @@
       "anyOf": [
         {
           "$ref": "../../../enums/vehicleSize.ignore.json"
-        },
-        {
-          "type": "null"
```
Comment (gjulien-bjss) [inline json-definitions/v3/tech-record/put/psv/complete/index.json:84]: same with the `anyOf` here

File: json-definitions/v3/tech-record/put/psv/complete/index.json [hunk 7]
```diff
@@ -147,9 +144,6 @@
           "type": "integer",
           "minimum": 0,
           "maximum": 99
-        },
-        {
-          "type": "null"
```
Comment (gjulien-bjss) [inline json-definitions/v3/tech-record/put/psv/complete/index.json:152]: same with the `anyOf` here

File: json-definitions/v3/tech-record/put/psv/testable/index.json [hunk 8]
```diff
@@ -128,9 +128,6 @@
           "type": "integer",
           "minimum": 0,
           "maximum": 99
-        },
-        {
-          "type": "null"
```
Comment (gjulien-bjss) [inline json-definitions/v3/tech-record/put/psv/testable/index.json:133]: same with the `anyOf` here

File: json-definitions/v3/tech-record/get/psv/complete/index.json (hunk 1 above)
Comment (gjulien-bjss) [reply json-definitions/v3/tech-record/get/psv/complete/index.json:85]: sorry about the bot 😢

**Pull request closed without merging**